frontend/src/app/libs/common-components-demo/components/input-demo/input-demo.component.html

```diff
@@ -0,0 +1,65 @@
+<app-demo-group class="input-demo" [title]="'Input'">
+	
+    <div class="light-theme input-demo__states">
+		<div class="input-demo__row">
+			<form [formGroup]="inputTextDemoForm">
+				<app-demo-state [title]="'Text'" [isDark]="false">
+					<app-input [formControl]="inputTextDemoForm.controls['text1']" [inputType]="'text'" [label]="'Your text without validate'" [placeholder]="'Your placeholder'" formControlName="text1"></app-input>
+					<app-input [formControl]="inputTextDemoForm.controls['text2']" [inputType]="'text'" [label]="'Your text with validate'" [placeholder]="'Your placeholder'" formControlName="text2"></app-input>
+					<app-input [formControl]="inputTextDemoForm.controls['text3']" [inputType]="'text'" [label]="'Your text'" [placeholder]="'Your placeholder'" formControlName="text3"></app-input>
+					<app-button type="flat" (click)="submit()" [disabled]="inputTextDemoForm.status==='INVALID'">Click</app-button>
+				</app-demo-state>
+			</form>
+
+			<form class="input-demo__row" [formGroup]="inputEmailDemoForm">
+				<app-demo-state [title]="'Email'" [isDark]="false">
+					<app-input [formControl]="inputEmailDemoForm.controls['email1']" [inputType]="'email'" [label]="'Your email'" [placeholder]="'Your placeholder'" formControlName="email1"></app-input>
+					<app-input [formControl]="inputEmailDemoForm.controls['email2']" [inputType]="'email'" [label]="'Your email'" [placeholder]="'Your placeholder'" formControlName="email2"></app-input>
+					<app-input [formControl]="inputEmailDemoForm.controls['email3']" [inputType]="'email'" [label]="'Your email'" [placeholder]="'Your placeholder'" formControlName="email3"></app-input>
```

Comment:
do we really need `formControlName` input? Seems like we already passing whole control

Reply:
also dont see `[value]` input  and `(valueChange)` output. We need to be able to use input in both ng form and just like component